components/gitpod-db/src/typeorm/entity/db-personal-access-token.ts

```diff
@@ -53,7 +53,7 @@ export class DBPersonalAccessToken {
     scopes: string[];
 
     @Column("datetime")
-    expirationTime: Date;
+    expirationTime: Date | null;
```

Comment:
AFAIR we've really never used `null` as value. Wouldn't setting a date in distant future be less likely to cause problems?

Reply:
@svenefftinge So just add option to allow 5 years? And not show `No expires!` but actual expiration date on dashboard?

<img width="1169" alt="SCR-20240117-dmpv" src="https://github.com/gitpod-io/gitpod/assets/20944364/c844c6dd-53cf-4f40-be68-41c84d47a9c4">

Reply:
Or hard code something like `2099-01-01 00:00:00` as never expires